app/code/Academy/ProductFeed/etc/db_schema.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0"?>
+<schema xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="urn:magento:framework:Setup/Declaration/Schema/etc/schema.xsd">
+    <table name="product_feed">
+        <column xsi:type="int" name="product_feed_id" padding="11"
+                unsigned="true" nullable="false" identity="true" comment="Product Feed ID"/>
+        <column xsi:type="varchar" name="name" nullable="false" length="255" comment="Product Feed Name"/>
+        <column xsi:type="varchar" name="filename" nullable="false" length="255" comment="Product Feed File Name"/>
+        <column xsi:type="varchar" name="file_type" nullable="false" length="10" comment="File format"/>
+        <column xsi:type="varchar" name="status" nullable="false" length="255" comment="Status"/>
+        <column xsi:type="varchar" name="FTP" nullable="true" length="255" comment="FTP address"/>
+        <column xsi:type="varchar" name="template_content" nullable="false" comment="Template content"/>
```

Comment:
Zamieniłabym varchar na text. Tutaj będzie cała treść templatki a to sporo tekstu wyjdzie.